### PR TITLE
Feature/score

### PR DESCRIPTION
**Pull request by vitor-paholsky**
1. add the proccesFindAll() func;
2. removed non needed imports;
3. switched to float32 instead of string on score struct.

### CODE REVIEW

File: server/http/category/score/score.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package score
+
+import (
+_ "encoding/json"
+"net/http"
+_ "net/http"
+_ "strings"
+"github.com/BorsaTeam/jams-manager/server"
```
Comment (victor-schumacher) [inline server/http/category/score/score.go:8]: why this ignored imports?

File: server/http/riders/score/score.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	return func(w http.ResponseWriter, r *http.Request) {
+		switch r.Method {
+		case http.MethodGet:
+			processFindAll(w)
+		case http.MethodPost:
+			processPost(w, r)
+		case http.MethodDelete:
+			processDelete(r)
+		default:
+			w.WriteHeader(http.StatusMethodNotAllowed)
```
Comment (victor-schumacher) [inline server/http/riders/score/score.go:27]: score.go needs to be only one file, the path must be /server/http/score/score.go

there's a similar example at /server/http/rider/rider.go

File: server/server.go [hunk 3]
```diff
@@ -19,6 +23,7 @@ type (
 		PaidSubscription bool     `json:"paidSubscription"`
 		Sponsors         []string `json:"sponsors"`
 		CategoryId       string   `json:"categoryId"`
+		Score            float32  `json:"score"`
```
Comment (kaduartur) [inline server/server.go:26]: Remove score from Rider and add RiderId to Score struct

File: server/http/score/score.go [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+	score := server.Score{}
+
+	defer r.Body.Close()
+
+	err := json.NewDecoder(r.Body).Decode(&score)
+	if err != nil {
+		w.Write([]byte("Errow while processing data"))
+		return
+	}
+	scoreId, err := m.createScore(score)
```
Comment (victor-schumacher) [inline server/http/score/score.go:48]: What you think about a space between 47 and 48?

File: server/database/repository/score_repository.go [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+	"github.com/BorsaTeam/jams-manager/server/database"
+)
+
+type Score interface {
+	Save(score ScoreEntity) (string, error)
+}
+
+type ScoreEntity struct {
+	Score    float32    `json:"score"`
+	Id       string     `json:"id"`
```
Comment (victor-schumacher) [inline server/database/repository/score_repository.go:16]: what about use omitempty, and generate a new id using uuid?

**Pull request closed without merging**